lang/en.json

```
@@ -106,7 +106,7 @@
 	"scoresVisibilityVisible": "Scores are visible to students",
 	"shareRubricSaveFailed": "Changing org unit availability failed",
 	"statistics": "Statistics",
-	"standardsAligned": "Standards aligned to {name}",
+	"standardsAligned": "{outcome, select, competencies {competencies} expectations {expectations} objectives {objectives} outcomes {outcomes} standards {standards} other {standards}} aligned to {name}",
```

```suggestion
	"standardsAligned": "{outcome, select, competencies {Competencies} expectations {Expectations} objectives {Objectives} outcomes {Outcomes} standards {Standards} other {Standards}} aligned to {name}",
```

These should be capitalized (will need to re-run lang scripts)